functions-methods.js

```diff
@@ -24,19 +18,7 @@ console.log(outcome)
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
-function typeOfEmail(email) {
-        if (email.includes("novi-education.nl")) {
-            console.log("Student");
-        } else if (email.includes("novi.nl")) {
-            console.log("Medewerker");
-        } else {
-            console.log("Extern");
-        }
-}
-
-const emailIncludes = typeOfEmail("rachelle.murk@novi-education.nl");
 
```

Comment:
Je kan ook het switch statement gebruiken. je kan ook de domain functie uit vorige opdracht gebruiken.
Staat ook in uitwerkingen.